alerts/openshift-virtualization-operator/NodeStatusMaxImagesExceeded.md

```diff
@@ -0,0 +1,34 @@
+# NodeStatusMaxImagesExceeded
+
+## Meaning
+
+This alert is applicable to clusters that have a large number of container
```

Comment:
I would remove the first sentence and change the second sentence to "This alert fires when a node exceeds..." for consistency.

Reply:
updated